dcos_launch/config.py

```diff
@@ -120,37 +123,54 @@ def get_validated_config(user_config: dict, config_dir: str) -> dict:
 
     # use the intermediate provider-validated config to add the platform schema
     platform = validator.normalized(user_config)['platform']
+    if provider == 'terraform' and 'ssh_user' in user_config['terraform_config']:
+        if platform in ('azure', 'aws'):
+            raise Exception('Cannot currently set ssh_user parameter for azure and gcp')
+        elif platform in ('gce', 'gcp'):
```

Comment:
How about having this as `if` and the `raise` above being in `else` with `platform` in the Exception message?

```
if platform in ('gce', 'gcp'):
  user_config['terraform_config']['gcp_ssh_user'] = user_config['ssh_user']
else:
  raise Exception('Cannot currently set ssh_user parameter for {}'.format(platform))
```

Reply:
Done